addons/web/static/src/views/relational_model.js

```diff
@@ -782,9 +782,10 @@ export class Record extends DataPoint {
      * @returns {boolean}
      */
     isReadonly(fieldName) {
+        const modelField = this.fields[fieldName];
         const activeField = this.activeFields[fieldName];
         const { readonly } = activeField.modifiers || {};
-        return readonly ? evalDomain(readonly, this.evalContext) : false;
+        return modelField.readonly || (readonly ? evalDomain(readonly, this.evalContext) : false);
```

Comment:
@sdegueldre I find it really strange that strict readonly fields (specified on python level) are not always considered readonly, independently of the modifiers specified in the view.

It could be caused by a recent change from DLE to avoid duplicating information into modifiers, but it should be fixed and considered IMHO (the same probably applies to the `invisible` attribute on fields, though not sure it makes any sense to specify a field `invisible=True` in the python code).

Reply:
The readonly field attribute in python is just the default value for the modifier and can be overriden inside the view, this is the expected behaviour and this change is incorrect. The python view processing code will set the readonly value specified for the field as the readonly modifier *unless* another is specified inline. This change is not correct.

Reply:
no, it should be possible to override a python ``````readonly in the view, the same way we allow overriding required

it should not be considered fixed

these are default values you set on the python field (same for domain, invisible, etc.), not mandatory ones - you may not like it, but there is no reason to change it for that :)

Reply:
Makes sense, but in this case, there is no modifiers in the view, so the readonly from the field should be applied.

Reply:
Edit: found the problem, not js side, but server side, i'll investigate a bit more then push the fix.